src/System.Threading.Tasks.Extensions/src/System.Threading.Tasks.Extensions.csproj

```diff
@@ -13,8 +13,7 @@
     <IsPackable>true</IsPackable>
     <VersionPrefix>4.5.4</VersionPrefix>
     <VersionPrefix Condition="'$(IsPackable)' == 'true'">4.6.0</VersionPrefix>
-    <AssemblyVersion Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.2.0.1</AssemblyVersion>
-    <AssemblyVersion Condition="'$(IsPackable)' == 'true' and $([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETFramework'">4.2.1.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(IsPackable)' == 'true'">4.2.1.0</AssemblyVersion>
```

Comment:
Confirmed that the inbox framework versions as `4.3.0.0`.
```
// C:\Users\ericstj\.nuget\packages\microsoft.netcore.app\2.1.0\ref\netcoreapp2.1\System.Threading.Tasks.Extensions.dll
// System.Threading.Tasks.Extensions, Version=4.3.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51
```

I think we can remove the comment on line 7 about being frozen -- we're  saying we'll allow this to increment (so long as it stays below 4.3.x.x.